Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -24,6 +30,18 @@ - (void)prepareForReuse {
     self.summaryLabel.text = nil;
 }
 
+-(void)awakeFromNib {
+    [self rememberSettingsFromIB];
+}
+
+-(void)rememberSettingsFromIB {
+    NSRange r = NSMakeRange(0, 1);
+    self.summaryLabelAttributesFromIB = [self.summaryLabel.attributedText attributesAtIndex:0 effectiveRange:&r];
+    self.paddingAboveDescriptionFromIB = self.paddingConstraintAboveDescription.constant;
+    self.paddingBelowDescriptionFromIB = self.paddingConstraintBelowDescription.constant;
+    self.heightOfImageFromIB = self.imageHeightConstraint.constant;
```

Comment:
For this one, can you just set the vertical content hugging priority to be greater than the priority of the imageHeightConstraint? I would think it would just collapse to 0 in this case.